packages/babel-plugin-strip-runtime/src/utils/ast.tsx

```diff
@@ -13,7 +13,7 @@ export const isCreateElement = (node: t.Node): node is t.CallExpression => {
   return (
     t.isMemberExpression(node) &&
     t.isIdentifier(node.object) &&
-    node.object.name === 'React' &&
+    node.object.name.startsWith('React') &&
```

Comment:
Can we leave a comment here talking through why we do "starts with" (because of different bundler outputs).

Reply:
Done